test/test-lib/fixtures.ts

```diff
@@ -448,6 +448,29 @@ const loaders: types.Dictionary<LoaderFunc> = {
 			user,
 		});
 	},
+	service_installs: async (jsonData, fixtures) => {
```

Comment:
Is this actually necessary? Service installs should always be automatically created whenever they're necessary and if they're not then I'd expect that's a bug that needs to be fixed

Reply:
@Page-  I think we might be able to have the service_install be created automatically but I think something in either the ordering of the mocks or maybe some missing property I could not figure it out yet causes it not to be created. I see that this is tested in cloud api more or less the same way so I followed from here: https://github.com/balena-io/balena-api/blob/master/test/20_state_endpoint.ts#L544-L553